proxy-libs/README.md

```diff
@@ -0,0 +1,13 @@
+# proxy-libs
+
+These are the supported libraries for auto instrumentation in go.
+
+Release supported tag for proxy-libs
+-
+``` git tag github.com/helios/go-sdk/proxy-libs/<PROXY_LIB_NAME>/<TAG> ```
```

Comment:
Tag does not include `github.com/helios/go-sdk/`, it's only `proxy-libs/heliosgrpc/v1.51.0`